CONTRIBUTING.md

```diff
@@ -41,6 +53,31 @@ We love pull requests, but would prefer that new contributors start with smaller
 
 It is also a good idea to add a comment to an issue that you are working on to let everyone know. If you stop working on it, also please let us know.
 
+### Pull request guidelines for
+
+#### Feature branches
+
+  1. Create a new branch named either `feature/*`, `bugfix/*` or `hotfix/*`
+  2. Before pull request, remember to update `CHANGELOG.md`
+     PS.If you are not releasing changes right away, add `Unreleased` instead of version number to `CHANGELOG.md`
+  3. Push feature branch, create pull request with a good name, and a comment if necessary
+  4. After approved review a maintainer will merge to `main`
+
+#### Release branches
+
+  1. Create a new branch named `release/x.x.x`
+  2. Change text `Unreleased` to `x.x.x` i `CHANGELOG.md`
+  3. Run `npm version [patch, minor, major]` to upgrade `package.json`
+  4. Push release branch and create pull request
+  5. After approved review a maintainer will merge to `main` (deploy)
```

Comment:
Should the pull requester be able to merge his/her own pull requests, as long as they have the necessary number of approved reviews and all build succeeds?

Reply:
Is that possible? Anonymous users from forks?